core/operation/src/main/java/uk/gov/gchq/gaffer/operation/Operation.java

```diff
@@ -28,7 +30,12 @@
  * Operations must be JSON serialisable in order to make REST API calls.
  */
 @JsonTypeInfo(use = JsonTypeInfo.Id.CLASS, include = JsonTypeInfo.As.PROPERTY, property = "class")
-public interface Operation {
+public interface Operation extends Closeable {
+
+    default void close() throws IOException {
```

Comment:
If there's no implementation here, maybe this is the wrong level at which to introduce this interface?